src/shared/treeview/nodes/awsCommandTreeNode.ts

```diff
@@ -23,4 +23,13 @@ export class AWSCommandTreeNode extends AWSTreeNodeBase {
         this.tooltip = tooltip
         this.contextValue = 'awsCommandNode'
     }
+
+    public determineRegion(): string {
+        if (this.parent) {
+            return this.parent.determineRegion()
+        } else {
+            // Not sure what to return here since it doesn't have a regionCode.
+            return 'None'
```

Comment:
can we return `this.regionCode` here?

Reply:
Unfortunately the `AWSTreeNodeBase` doesn't have regionCode. The attribute is added in some of the classes that inherit from it in their constructors. The way around this I thought of was the `determineRegion` method, but is there a better way?